views/components/ChartList.jsx

```diff
@@ -0,0 +1,72 @@
+import React, { useEffect, useState } from 'react'
+
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
+import Button from '@material-ui/core/Button';
+import Delete from '@material-ui/icons/Delete';
+
+import { getCharts, deleteChart } from '../fe-utils/fetchUtil'
+
+const ChartList = () => {
+  const [chartList, setChartList] = useState([])
+
+  useEffect(() => {
+    getCharts().then((res) => setChartList(res.data))
+  }, [])
```

Comment:
❓ Is there already some sort of loading dialog to let the user know that the chart is loading? In case they're on a slow connection.

Reply:
This is a good idea for future iterations, I'll make an issue for it